src/interaction/EfInteraction.py

```
@@ -34,6 +34,9 @@ def __init__(self, *args, **kwargs):
         self.cursor_position = None
         self.activated = False
         self._esc_hwnd = 0
+        self._key_prev_hwnd = 0  # 后台模式下按键按下前的前台窗口，松开时恢复
+        self._background_key_hold_count = 0  # 后台模式下未释放的按键数
+        self._pressed_keys = set()  # 已成功按下的按键（用于配对保护）
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)